File: Row_with_max_1s/code.cpp
//User function template for C++
class Solution{
public:
	int rowWithMax1s(vector<vector<int> > arr, int n, int m) {
	    vector<int> temp;
	    int mx = 0;
	    int res = -1;
	    int count = 0;
	    for(int i=0; i<arr.size(); i++){
	        temp = arr[i];
	        count = 0;
	        for(int k=0; k<temp.size(); k++){
	            if(temp[k] == 1){
	                count++;
	            }
	        }
	        if(count > mx){
	            mx = count;
	            res = i;
	        }
	    }
	    return res;
	}

};

File: Row_with_max_1s/code.java
//User function Template for Java

class Solution {
    int rowWithMax1s(int arr[][], int n, int m) {
        int c=0,max=0,res=-1;
        for(int i=0;i<n;i++)
        {
            c=0;
            for(int j=0;j<m;j++)
            {
                if(arr[i][j]==1)
                c++;
                if(c>max)
                {
                    max=c;
                    res=i;
                }
            }
        }
        return res;
    }
}
